frontstage/views/account/account_transfer_survey.py

```diff
@@ -309,22 +272,25 @@ def send_transfer_instruction(session):
     party_id = session.get_party_id()
     respondent_details = party_controller.get_respondent_party_by_id(party_id)
     if form["email_address"].data != email:
-        raise ShareSurveyProcessError("Process failed due to session error")
+        raise TransferSurveyProcessError("Process failed due to session error")
```

Comment:
Process failed due to session error, not sure what this means

Reply:
Updated to something more meaningful.